packages/graphql-mini-transforms/src/webpack.ts

```diff
@@ -20,8 +28,19 @@ export default async function graphQLLoader(
   }
 
   try {
+    const cleanDocumentOptions = {} as CleanDocumentOptions;
+
+    if (options) {
+      cleanDocumentOptions.generateId = options.generateId;
+    }
```

Comment:
We generally use ternaries over mutating objects `const cleanDocumentOptions = {generateId: options ? option.generateId, undefined}`